Makefile

```diff
@@ -1,2 +1,33 @@
+# A Self-Documenting Makefile: http://marmelab.com/blog/2016/02/29/auto-documented-makefile.html
+
+VERSION=$(shell git describe --always --tags | cut -d "v" -f 2)
+LINKER_FLAGS=-s -w -X github.com/tufin/oasdiff/build.Version=${VERSION}
+GOLANGCI_VERSION=v1.50.1
```

Comment:
- I see that the latest stable version is 1.52.2
- I prefer to call it `GOLANGCILINT_VERSION`

Reply:
Updated
```bash
make devtools                                                                                                                                                 1.20.3  16.18.0 bazel 5.4.0
==> Installing dev tools...
curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b /Users/andrea.angiolillo/go/bin v1.52.2
golangci/golangci-lint info checking GitHub for tag 'v1.52.2'
golangci/golangci-lint info found version: 1.52.2 for v1.52.2/darwin/arm64
golangci/golangci-lint info installed /Users/andrea.angiolillo/go/bin/golangci-lint
```